FRINXio/frinx-frontend
# add frontend server package; update webpack config; remove ngnix stuff

We need to be able to set `window.__CONFIG__` values via environment variables. I created a simple server thats doing two things:
- in `development` proxies all the request to `webpack-dev-server` (therefore "nothing" will change)
- in `production` it will serve all the static files (it replaces previous ngnix server)
It also (most importantly) serves config values set by the environment (`/-/config.js`) - it replace old `.shtml` solution.
To test if everything is working correctly:
- run `cp packages/frinx-frontend-server/.env.example packages/frinx-frontend-server/.env`
- run `yarn run dev` in the root of the repository
- open `:3000` in your browser - you should see UI
- (optionally) start `ui-dev-proxy` (`npm run start:fm`)
- open `:4000` in your browser - everything should work as before

Dockerfile changes are subject to changes based on feedback from @Jozefiel
